spec/lib/administrate/page/collection_spec.rb

```diff
@@ -0,0 +1,172 @@
+require "active_support/core_ext/module"
+require "administrate/page/collection"
+
+describe Administrate::Page::Collection do
+  let(:scopes_hash) do
+    {
+      :symbol_key => [ :active, :inactive, "active_since(1992)" ],
+      "string_key" => [ "funny", :sad ]
```

Comment:
Space inside square brackets detected.<br>Put a comma after the last item of a multiline hash.